studio/pages/new/[slug].tsx

```diff
@@ -467,20 +467,26 @@ const Wizard: NextPageWithLayout = () => {
                   description={
                     <div className="space-y-3">
                       <p className="text-sm leading-normal">
-                        This organization uses the new organization-level billing and is on the{' '}
+                        This organization uses the new organization-level-billing and is on the{' '}
                         <span className="text-brand">{orgSubscription?.plan?.name} plan</span>.
                       </p>
 
                       {/* Show info when launching a new project in a paid org that already has at least one project */}
                       {orgSubscription?.plan?.id !== 'free' && orgProjectCount > 0 && (
-                        <p>
-                          Launching another project incurs additional compute costs (starting at $10
-                          per month).
-                        </p>
+                        <div>
+                          <p>
+                            Launching another project incurs additional compute costs, starting at
+                            $0.01344 per hour (~$10/month).
+                          </p>
+                          <p>
+                            If you have not exceeded your 2 free project limit, you can also create
```

Comment:
humm reverse this maybe?

You can also create a new Free plan organization if you have not exceeded your 2 free project limit. 

But that's a bit clunky too — also inconsistent use of free vs Free. 

— 
maybe: 
You can also create a new organization under the free plan (unless you have exceeded your 2 free project limit).

Reply:
Using your last suggestion